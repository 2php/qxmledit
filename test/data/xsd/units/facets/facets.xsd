<?xml version="1.0" encoding="UTF-8" ?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">

<xs:simpleType name="xxx">
    <xs:restriction base="xs:string">
      <xs:pattern value="([a-z])*"/>
      <xs:minInclusive value="1"/>
      <xs:maxInclusive value="2"/>
      <xs:minExclusive value="3"/>
      <xs:maxExclusive value="4"/>
      <xs:totalDigits value="5"/>
      <xs:fractionDigits value="6"/>
      <xs:totalDigits value="5"/>
      <xs:length value="8"/>
      <xs:minLength value="10"/>
      <xs:maxLength value="11"/>
      <xs:whiteSpace value="preserve"/>
    </xs:restriction>
</xs:simpleType>

</xs:schema>
