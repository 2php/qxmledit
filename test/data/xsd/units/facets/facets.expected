<li>pattern: <B>([a-z])*</B></li>
<li>minInclusive: <B>1</B></li>
<li>maxInclusive: <B>2</B></li>
<li>minExclusive: <B>3</B></li>
<li>maxExclusive: <B>4</B></li>
<li>totalDigits: <B>5</B></li>
<li>fractionDigits: <B>6</B></li>
<li>length: <B>8</B></li>
<li>minLength: <B>10</B></li>
<li>maxLength: <B>11</B></li>
<li>whiteSpace: <B>preserve</B></li>
